src/articles/playbooks/guides/index.md

```diff
@@ -20,6 +20,6 @@ Well, in this set of guides, we provide a set of practical Playbooks you can sta
 
 // TODO
 
-## [Customer Usage Stats](./sharing-usage-stats.md)
+## [Customer Usage Stats](./sharing-usage-report.md)
```

Comment:
I don't really know that I like using headings for these. I think it should just be a bulleted list of guides.